src/trusted/service_runtime/lind_syscalls.c

```diff
@@ -745,7 +745,7 @@ StubType stubs[] = {
         {LindCommonPreprocess, 0, 0}, /* 42 LIND_safe_net_getsockname */
         {LindCommonPreprocess, 0, 0}, /* 43 LIND_safe_net_getsockopt */
         {LindCommonPreprocess, 0, 0}, /* 44 LIND_safe_net_setsockopt */
-        {LindCommonPreprocess, 0, 0}, /* 45 LIND_safe_net_shutdown */
+        {0},
```

Comment:
Keeping the comment just to `/* 45 */` would probably be preferable

Reply:
Added `/* 45 */`